.storybook/main.js

```diff
@@ -36,7 +36,9 @@ module.exports = /** @type {Omit<StorybookConfig,'typescript'|'babel'>} */ ({
     '@storybook/addon-knobs/preset',
     'storybook-addon-performance',
     'storybook-addon-export-to-codesandbox',
-    '@fluentui/react-storybook-addon',
+
+    // if addon is not installed from npm it needs to be registered via absolute or relative path. otherwise it wont work
+    require.resolve('@fluentui/react-storybook-addon').replace('lib-commonjs/index.js', 'preset.js'),
```

Comment:
require.resolve('@fluentui/react-storybook-addon') returns lib-commonjs\index.js on windows!

Reply:
```suggestion
     path.normalize(require.resolve('@fluentui/react-storybook-addon') + '/../../preset.js'),
```

Reply:
just curious if you're using cmd.exe or wsl has a bug. anyways good catch

Reply:
i was using git bash